airbyte-integrations/connectors/source-bigquery/source_bigquery/streams.py

```diff
@@ -59,7 +65,16 @@ def next_page_token(self, response: requests.Response, **kwargs) -> Optional[Map
         return None
 
     def process_records(self, record) -> Iterable[Mapping[str, Any]]:
```

Comment:
Can we rename `record` here to something more specific? I would guess this is a `table_description` or something like that

Reply:
I think here it's better for `BigqueryStream` to have a general name since the record type will depend on the `path` set which is set dynamically here.